### ISSUE TITLE
contribution guidelines not in sync with the rest of jquery

### ISSUE THREAD

**Issue by toejough** [jquery/learn.jquery.com#554]
The contribution guidelines (https://github.com/jquery/learn.jquery.com/blob/master/CONTRIBUTING.md) are not in sync with other jquery projects (such as https://github.com/jquery/jquery.com/blob/master/CONTRIBUTING.md) and don't point you towards the overall guidelines (here: http://contribute.jquery.org/).

This means that first-time contributors have no idea how to do things correctly... :-/

**Comment by RedWolves** [comment]
I'd like to propose that we rename contributing.md to contribute.md 

and then create a CONTRIBUTING.md with the proper guidelines for the repo.

The confusion is that contributing.md is an actual top level page on the learn site https://learn.jquery.com/contributing/

cc. @scottgonzalez

**Comment by arthurvr** [comment]
> I'd like to propose that we rename contributing.md to contribute.md

Nah. Let's try to not confuse people. In fact the current content in `contributing.md` would need to be merged with the content on `contribute.jquery.org`, and the link in our header should link to there. This way we're duplicating content and indeed aren't really friendly for first-time contributors.

**Comment by scottgonzalez** [comment]
> I'd like to propose that we rename contributing.md to contribute.md

If you want to do that, you can, it's not really related to this issue.

> and then create a CONTRIBUTING.md with the proper guidelines for the repo.

This file already exists, but has bad content, which is what this issue is about.

> The confusion is that contributing.md is an actual top level page on the learn site https://learn.jquery.com/contributing/

If you're referring to this issue and the preceding confusion and problems, this is completely unrelated. The problem is that the content is bad and out of sync with the standard contribution process.

If you're referring to some separate confusion about the relationship between the two, the two page are technically separate, they're only linked because @ajpiano decided to do so.

**Comment by arthurvr** [comment]
@scottgonzalez @RedWolves I opened two PRs to fix this #634 and jquery/jquery-wp-content#343. Mind giving those a review?
